src/index.d.ts

```diff
@@ -275,9 +275,19 @@ export namespace h {
 // Preact render
 // -----------------------------------
 
+interface ContainerNode {
+	nodeType: Node['nodeType'];
+	parentNode: Node['nodeType'];
```

Comment:
I think this is a typo. Should this be
```suggestion
	parentNode: Node['parentNode'];
```

I think that'll fix the tests. Could we also add additional tests in that same file `test/ts/preact.tsx` for the additional things preact supports mounting into? Something like:
```ts
render(h('div', {}), document.createElement('div'));
render(h('div', {}), document);
render(h('div', {}), document.createElement('div').shadowRoot!);
render(h('div', {}), document.createDocumentFragment());
```
```

Reply:
Applied both changes, will watch to see if CI passes now.

Reply:
Looks like I have to wait for approval before workflow will run.

Reply:
Lol just realized the typo came from my original comment 😅 Whoops! Thanks for applying the fix